src/views/product-integration/component-view/components/searchable-variable-group-list/style.module.css

```diff
@@ -19,10 +19,10 @@
 }
 
 .results {
-	color: var(--gray-3);
+	composes: hds-typography-body-200 from global;
```

Comment:
Related to this ticket https://app.asana.com/0/1203792616809419/1203988523494882/f